### PR TITLE
add documentation for AssetScanner

### PR DESCRIPTION
**Pull request by mrtraser**

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-119.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
Aren't there any required props for the scanner 🤔 ?

File: src/components/AssetScanner/stories/AssetScanner.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import { storiesOf } from '@storybook/react';
+import { message } from 'antd';
+import React from 'react';
+import { ErrorResponse } from '../../../interfaces';
+import { ASNotifyTypes, AssetScanner } from '../AssetScanner';
+
+import * as full from './full.md';
+import * as customNotificationsDoc from './notifications.md';
+
+const onUnauthorized: any = (error: ErrorResponse) => {
```
Comment (f1cognite) [inline src/components/AssetScanner/stories/AssetScanner.stories.tsx]:
```suggestion
const onUnauthorized: (error: ErrorResponse) => {
```

File: src/components/AssetScanner/stories/full.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  );
+  
+}
+```
+
+#### Available props:
+
+##### Required:
+
+_No required props_
```
Comment (f1cognite) [inline src/components/AssetScanner/stories/full.md]: Really no one?

File: src/components/AssetScanner/stories/full.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+##### Optionals:
+
+| Property             | Description                                                                | Type                             | Default                                          |
+| -------------------- | -------------------------------------------------------------------------- | -------------------------------- | ------------------------------------------------ |
+| `ocrUrl`             | URL of detect service (OCR) API                                            | `string`                         | https://vision.googleapis.com/v1/images:annotate |
+| `ocrKey`             | Key property for OCR service                                               | `{ [name: string]: string }`     |                                                  |
+| `customNotification` | Function which is react on provided type of notification                   | `(type: ASNotifyTypes) => any`   |                                                  |
```
Comment (f1cognite) [inline src/components/AssetScanner/stories/full.md]:
```suggestion
| `customNotification` | Callback function to react on provided type of notification                   | `(type: ASNotifyTypes) => any`   |                                                  |
```

File: src/components/AssetScanner/stories/full.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+| `onStringRecognize`  | Callback which is triggering when strings was recognised on image          | `(strings: string[]) => void;`   |                                                  |
```
Comment (f1cognite) [inline src/components/AssetScanner/stories/full.md]:
```suggestion
| `onStringRecognize`  | Callback which is triggering when strings was recognised in image          | `(strings: string[]) => void;`   |                                                  |
```

File: src/components/AssetScanner/stories/full.md (hunk 2 above)
Comment (mrtraser) [reply src/components/AssetScanner/stories/full.md]: users can use scanner without providing any props, but they might not get expected behaviour without providing some of optional params. In storybook they can find description of all props with some examples

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-119.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-119.gearboxjs.preview.cogniteapp.com

**Pull request merged**